CoreDependencies/src/ivy.xml

```diff
@@ -69,7 +69,7 @@
         <dependency org="com.fasterxml.jackson.core" name="jackson-databind" rev="2.10.3"/>
         <dependency org="com.fifesoft" name="rsyntaxtextarea" rev="3.1.1"/>
         <dependency org="gov.nasa" name="worldwind" rev="2.0.0"/>
-        <!--<dependency org="javax.help" name="javahelp" rev="2.0.05"/>-->
+        <dependency org="javax.help" name="javahelp" rev="2.0.05"/>
```

Comment:
@cygnus-x-1 in PR #504 I've got the new help system ready for testing. Everything worked with vanilla NB 12 without needing this help jar so please remove it from your PR.

Reply:
Are you able to remove this dependency @cygnus-x-1 so I can merge it in as discussed?

Reply:
@cygnus-x-1 I'm going to merge this and then revert this line.